packages/core/addon/models/bard-request-v2/request.js

```diff
@@ -110,4 +119,280 @@ export default class Request extends Fragment.extend(Validations) {
       dataSource: clonedRequest.dataSource
     });
   }
+
+  /**
+   * @property {ColumnFragment} timeGrainColumn - The column containing the dateTime time-dimension
+   */
+  @computed('columns.[]')
+  get timeGrainColumn() {
+    return this.columns.find(column => column.type === 'time-dimension' && column.field === 'dateTime');
```

Comment:
Should this be 

`column.type === 'time-dimension'` OR `column.field === 'dateTime'`

Reply:
Should this be set of time-dimension?

Reply:
We have time-dimensions that aren't timegrain columns.